### ISSUE TITLE
How can I change the loss function of the classification model

### ISSUE THREAD

**Issue by spaul13** [AlexeyAB/darknet#4911]
In general, loss is the L2 distance between the actual label and the predicted label in N-dimensional space (where N is the number of labels). And that loss backpropagated during the gradient update.

But I want to change the loss function can anyone plz tell me how to achieve that?

What I observed after forward_network() and backward_network() there is a get_network_cost(). Do I need to change that? Any pointers on this will be highly helpful.

**Comment by WongKinYiu** [comment]
default loss is softmax with cross entropy loss.
https://github.com/AlexeyAB/darknet/blob/master/src/softmax_layer.c#L109-L117

**Comment by spaul13** [comment]
@WongKinYiu how can I change the loss function in this repo?

**Comment by WongKinYiu** [comment]
replace `softmax_x_ent_gpu` and `softmax_x_ent_cpu` to your custom loss function.
